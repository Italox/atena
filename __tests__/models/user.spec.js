import User from "../../models/user";

describe("[Models] User", () => {
  it("should be invalid if name is empty", () => {
    let user = new User();
    user.validate(err => {
      expect(err.errors.name).toBeTruthy();
    });
    expect(user).toMatchSnapshot();
  });
});
