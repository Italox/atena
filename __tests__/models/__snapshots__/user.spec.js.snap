// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Models] User should be invalid if name is empty 1`] = `
Object {
  "_id": "5be5e5efba5f4212fe473bcd",
  "isCoreTeam": false,
  "lastUpdate": 2018-11-09T19:54:23.960Z,
}
`;
